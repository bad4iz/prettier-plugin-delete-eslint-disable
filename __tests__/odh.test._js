const { format } = require('prettier')

/**
 * @param {string} code
 * @param {prettier.Options} [options]
 */
const prettify = (code, options) =>
    format(code, {
        plugins: ['.'],
        filepath: 'file.ts',
        deleteEslintDisable: true,
        sortingImports: true,
    })

describe('file odh.spec', () => {
    it.only('should ', () => {
        expect.hasAssertions()
        // Arrange (всякие моки)

        // Act
        const res = prettify(`
import {
  CREATE_TYPE_NAME,
  CreateTypeRF,
  IsDiffHeightMarkRF,
  NumberFieldRF,
} from 'core/form/reduxForm/fields';
`)
        // Assert
        expect(res).toMatchSnapshot()
        expect(res).toBe(
            'import {\n' +
                '  CREATE_TYPE_NAME,\n' +
                '  CreateTypeRF,\n' +
                '  IsDiffHeightMarkRF,\n' +
                '  NumberFieldRF,\n' +
                '} from "core/form/reduxForm/fields";\n'
        )
    })
})
