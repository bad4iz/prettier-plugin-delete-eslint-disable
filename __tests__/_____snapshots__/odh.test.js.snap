// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`file odh.spec should  1`] = `
"import {
  CREATE_TYPE_NAME,
  CreateTypeRF,
  IsDiffHeightMarkRF,
  NumberFieldRF,
} from "core/form/reduxForm/fields";
"
`;
